qiskit/circuit/library/standard_gates/multi_control_rotation_gates.py

```diff
@@ -25,12 +25,14 @@
 
 def _apply_cu3(circuit, theta, phi, lam, control, target, use_basis_gates=True):
```

Comment:
the `mcrx`, `mcry` and `mcrz` gates in this file still call `_apply_cu3` and `_apply_mcu3_graycode` (which are still implemented using U1Gate and U3Gate). Can you update this whole file to use the new gates?

Reply:
I'm a little afraid that this could lead to performance issues, as these gates are called a lot in `control` and we don't have the 1-qubit optimizers ready for P/U yet, right? We'll have to check if this slows Qiskit down.